MaterialDesignThemes.Wpf/DataGridTextColumn.cs

```diff
@@ -15,6 +15,25 @@ public class DataGridTextColumn : System.Windows.Controls.DataGridTextColumn
             return null;
         }
 
+        protected override FrameworkElement? GenerateElement(DataGridCell cell, object dataItem)
+        {
+            var element = base.GenerateElement(cell, dataItem);
+
+            if (
+                element is TextBlock t
+                && cell.Column is System.Windows.Controls.DataGridTextColumn col
+                && DataGridAssist.GetElementStringFormat(col) is string stringFormat
+                && t.GetBindingExpression(TextBlock.TextProperty)?.ParentBinding is Binding binding
+                )
+            {
+                var NewBinding = binding.Clone();
+                NewBinding.StringFormat = stringFormat;
+                t.SetBinding(TextBlock.TextProperty, NewBinding);
+            }
+
+            return element;
+        }
```

Comment:
Although this works, I can imagine a similar override being needed in other column types if the attached DP (`ElementStringFormat`) should be leveraged in those.

For this reason, I think it may be cleaner to create a dedicated column type which encapsulates this type of functionality.

Reply:
Fixed in latest commit

Reply:
Don't worry about implementing any of this, I just wanted to show what I did to get my own project working and provide ideas for future development if a similar topic ever comes up. If you use/modify any or none of this code, no problem.

Thank you very much for your input!